docs/pact-functions.md

```diff
@@ -228,6 +242,18 @@ pact> (list 1 2 3)
 List modules available for loading.
 
 
+### make-list {#make-list}
```

Comment:
For naming Pact looks to LISP vs Haskell for naming precedents, thus `constantly`, `identity`, `make-list`. Within reason of course: `map` instead of `mapcar` for instance